core/src/main/scala/filodb.core/query/ResultTypes.scala

```diff
@@ -27,10 +27,12 @@ final case class ColumnInfo(name: String, colType: Column.ColumnType)
 /**
  * Describes the full schema of result types, including how many initial columns are for row keys.
  * The first ColumnInfo in the schema describes the first vector in Vectors and first field in Tuples, etc.
- * @param brSchemas if any of the columns is a binary record, thsi
+ * @param brSchemas if any of the columns is a BinaryRecord: map of colNo -> inner BinaryRecord schema
+ * @param fixedVectorLen if defined, each vector is guaranteed to have exactly this many output elements
```

Comment:
Not clear to me in what situations it is defined and not defined? Can we document known scenarios.

Reply:
ok